scapy/layers/l2.py

```diff
@@ -143,20 +167,23 @@ def i2h(self, pkt, x):
                 else:
                     x = "ff:ff:ff:ff:ff:ff"
                     warning("Mac address to reach destination not found. Using broadcast.")  # noqa: E501
-        return MACField.i2h(self, pkt, x)
+        return cast(str, super(DestMACField, self).i2h(pkt, x))
```

Comment:
It surprises me that this cast is necessary

Reply:
If I just use super() but without the cast I get an error: `Returning Any from function declared to return "str"`.
With the original unbound method call I also get:
```
Argument 1 to "i2h" of "Field" has incompatible type "DestMACField"; expected "Field[I, M]"
Argument 3 to "i2h" of "Field" has incompatible type "Optional[str]"; expected "I"
```